### PR TITLE
Next hit

### PR DESCRIPTION
**Pull request by henryaddison**
Get Ryan to look at it

### CODE REVIEW

**Comment by RyanGosling** [comment]
Hey girl!

**Comment by henryaddison** [comment]
Hey!
Could you take a look at XVIII - any suggestions?

File: XVIII [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Shall I compare thee to a Wednesday?
+Thou art more lovely and more temperate:
+Rough winds do shake the darling buds that day,
+And mid-week's lease hath all too short a date:
+Sometime too hot the eye of heaven shines,
+And often is his gold complexion dimmed,
```
Comment (RyanGosling) [inline XVIII]: My complexion's not gold!

**Comment by RyanGosling** [comment]
Much better!

Cheers Henners!

**Pull request merged**